### DISCUSSION TITLE
Does anyone know why NebulaLogger is not available on the AppExchange?

### DISCUSSION THREAD

**Discussion by mrjcrm** [jongpie/NebulaLogger#762] [General]
Does anyone know why NebulaLogger is not available on the AppExchange?

**Comment by jongpie** [comment]
@50471736 it's come up a few times (mostly in issue #290, including [this comment](https://github.com/jongpie/NebulaLogger/issues/290#issuecomment-2318830252)) - essentially, I have 2 concerns:
1. Having it on AppExchange would interfere with some functionality related to tracking platform limits
2. Going through the security review process is not something that I really look forward to doing / it might slow down my ability to quickly release new versions

I'm still leaning towards not publishing it on AppExchange, but I'll revisit the idea sometime next year (after I've wrapped up a few bigger enhancements that I have planned)
